src/Director/BodyTo/BodyToOrder.php

```diff
@@ -21,20 +21,24 @@ class BodyToOrder
      */
     public static function build(array $data): Order
     {
-        $order = (new OrderInternal(false))
+        $order = new OrderInternal(
+            allowEmptyRelations: false
+        );
+
+        $order
             ->setCreatedAt(self::date($data, 'created_at'))
             ->setDeletedAt($data['deleted_at'] ? self::date($data, 'deleted_at') : null)
             ->setFirst($data['is_first'])
-            ->setHidden($data['is_hidden'])
             ->setId($data['id'])
             ->setInvoiceNumber($data['invoice_number'])
             ->setInvoiceStatus(InvoiceStatus::from($data['invoice_status']))
-            ->setMode(Mode::from($data['mode']))
             ->setReference($data['reference'])
             ->setSource(Source::tryFrom($data['source'] ?? '') ?? Source::UNKNOWN)
+            ->setUpdatedAt(self::date($data, 'updated_at'))
+            ->setHidden($data['is_hidden'])
+            ->setMode(Mode::from($data['mode']))
             ->setAmount((float) $data['amount'])
-            ->setAmountWithTax((float) $data['amount_with_tax'])
-            ->setUpdatedAt(self::date($data, 'updated_at'));
+            ->setAmountWithTax((float) $data['amount_with_tax']);
```

Comment:
Het maakt mij niet heel veel uit, maar eerder was dit op alfabetische volgorde.

Reply:
De rede dat ik dit heb verplaatst is omdat ik in de IDE meldingen kreeg dat het niet bestaat ik denk dat dat te maken heeft met het gebruikt van `/** @internal */`? Correct me if I am wrong. Maar nu heb ik dat totaal niet meer.

Als ik nu bijvoorbeeld op `main` branch kijk heb ik die error meldingen terwijl de method `setId` wel bestaat. 

![CleanShot 2024-08-12 at 17 19 49@2x](https://github.com/user-attachments/assets/48df8c92-9403-41b7-b960-e939a29256e3)

Reply:
Het is wel heel gek, als de methods dus in de juiste volgorde plaats heb ik deze meldingen niet, en mijn VSC en plugins zijn allemaal up to date